docs/cosa-significa-essere-un-dev/README.md

```diff
@@ -0,0 +1,97 @@
+
+La verità, caro lettore, è che essere un dev significa essere tante cose contemporaneamente.
+
+Tante _definizioni_, se così vogliamo chiamarle, che però affondano le loro radici su dei principi solidi e genuini: [**passione**](#la-passione-il-combustibile-per-eccellenza), [**curiosità**](#la-curiosità-il-potere-della-scoperta), [**empatia**](#lempatia-la-capacità-di-impersonare-il-prossimo) e [**contaminazione**](#la-contaminazione-fare-squadra-nel-modo-giusto); principi su cui, peraltro, nasce questo libro.
+
+Caratteristiche che nulla hanno a che fare con la scrittura del codice, né presentano alcun tipo di legame diretto con specifiche competenze tecniche; nonostante ciò, risultano essere caratteristiche **fondamentali** per intraprendere questa carriera al meglio, lasciarsi coinvolgere da essa e riuscire a portare valore in questo settore.
+
+## La passione: il combustibile per eccellenza
+
+Decidere di "diventare un dev" significa scegliere di "sacrificare" buona parte del proprio tempo al fine di raggiungere un'obiettivo; non sarà un percorso facile, tantomeno breve; ci saranno momenti di eccitazione, ma anche di sconforto; ci saranno giorni in cui tutto sembrerà andare per il verso giusto, ed altri in cui ogni nostra singola attività verrà minata dalle difficoltà che ci si pareranno davanti; ci saranno concetti che domeremo con estrema facilità, ed altri in cui ne usciremo sconfitti e con la coda tra le gambe.
+
+Fà tutto parte del gioco: _abbraccialo, comprendilo e fallo tuo_; magari non oggi e forse nemmeno domani, ma arriverà il momento in cui riuscirai dove prima hai fallito.
+
+Solitamente, quando ci appassioniamo a qualcosa tendiamo a dedicare molto del nostro tempo a quella specifica attività; la coltiviamo negli anni prendendocene cura, approfondendo ogni aspetto e cercando di scoprire ogni singola caratteristica che possiede...in buona sostanza: _la rendiamo nostra_.
+
+Forti di questa consapevolezza possiamo quindi affermare che ciò che farà la differenza tra il nostro successo e la nostra sconfitta sarà la passione che metteremo in ciò che facciamo; se non avremo passione, difficilmente riusciremo ad accettare una sconfitta e a rialzarci più forti di prima; senza passione non riusciremo a trovare la forza di studiare, magari dopo una giornata di lavoro.
+
+Se però avremo passione, avremo dalla nostra una risorsa sorprendente: il dono della _Perseveranza_.
+
+Non importerà quindi quante volte proveremo, senza successo, a completare un'esercizio o ad imparare una tecnologia: avremo sempre la forza necessaria per incassare il colpo, prendere fiato, e provare a superare noi stessi.
+
+Il mio consiglio per te caro lettore, è quello di tenere a mente, per l'intera durata di questo manoscritto, il ruolo che ricopre la passione riguardo al percorso che hai scelto di intraprendere: se avrai passione, avrai la forza di perseverare.
```

Comment:
Toglierei "per l'intera durata di questo manoscritto". La frase mi piace molto, ma non sappiamo ancora in che punto del libro sarà questo contenuto. Inoltre, gli argomenti sono vari e disparati quindi non è detto vengano letti nell'ordine che ci aspettiamo. Se questo capitolo fosse, che ne so, il penultimo, la frase assumerebbe poco senso.

Reply:
"caro lettore" sostituibile con "che leggi"